.circleci/config.yml

```diff
@@ -7,7 +7,7 @@ orbs:
 executors:
   arlo:
     docker:
-      - image: cimg/python:3.9.5-browsers
```

Comment:
This image had an old version of poetry, which didn't work for us since we upgraded the dev env to use the latest poetry